### DISCUSSION TITLE
Results from dependency check and track

### DISCUSSION THREAD

**Discussion by bhuvi11** [DependencyTrack/dependency-track#1266] [Q&A] [answered]
Hello Team,

I got result for an MAVEN project from dependency track and check and i see similar vulnerabilities found.
If i am able to get the vulnersbilities from dependency check , wat should i expect from track ?

**Comment by msymons** [comment, marked as answer]
This is the technical comparison between the two:

https://docs.dependencytrack.org/odt-odc-comparison/

The bottom line is that the fuzzy matching that is performed by Dependency Check does tend to result in a lot of false positives (FP) that will not show up in Dependency-Track.

Note: that does not mean that FPs won't ever show up in DT.  If the data is incorrect in OSS-Index then that can lead to FPs in DT.  OSS-Index does allow such problems to be reported via [vulns](https://github.com/OSSIndex/vulns) and they are good about fixing things.  However,  DT does not automatically clean-up the "old" FP data (but will be correct for any new project analysis).  This is where you should be using the "Suppress" option in Audit.
